src/sentry/features/__init__.py

```diff
@@ -107,6 +107,7 @@
 default_manager.add("organizations:related-events", OrganizationFeature)  # NOQA
 default_manager.add("organizations:relay", OrganizationFeature)  # NOQA
 default_manager.add("organizations:reprocessing-v2", OrganizationFeature)  # NOQA
+default_manager.add("organizations:required-email-verification", OrganizationFeature)  # NOQA
```

Comment:
Should this be a flagr flag? It would need a trailing True for that.

Reply:
👍